lib/image.coffee

```diff
@@ -12,6 +12,10 @@ emit = ($item, item) ->
 
 bind = ($item, item) ->
   $item.dblclick -> editor.textEditor $item, item
-  $item.find('img').dblclick -> dialog.open item.text, this
+  url = item.url
+  if item.ipfs?
+    $.get "http://localhost:8080/ipfs/#{item.ipfs}", ->
```

Comment:
This is likely to be slow. The Firefox plugin checks for the local gateway by polling it using the IPFS API.